src/index.js

```diff
@@ -1,14 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { BrowserRouter as Router } from 'react-router-dom';
 import App from './components/App';
 import './index.css'
 
 ReactDOM.render(
     <React.StrictMode>
-        <Router>
-            <App />
-        </Router>
+        <App />
```

Comment:
ahh youre moving it from this file into the App component. Is there a reason we need to move our Router out of this file into the App.js?